src/Table/TableFooter.js

```diff
@@ -0,0 +1,85 @@
+// @flow
+
+import React from 'react';
+import type { ComponentType, Node } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import withStyles from '../styles/withStyles';
+
+export const styles = (theme: Object) => ({
+  root: {
+    fontSize: 12,
+    color: theme.palette.text.secondary,
+  },
+});
+
+type DefaultProps = {
+  classes: Object,
+  component: string,
+};
+
+export type Props = {
+  /**
+   * The content of the component, normally `TableRow`.
+   */
+  children?: Node,
+  /**
+   * Useful to extend the style applied to components.
+   */
+  classes?: Object,
+  /**
+   * @ignore
+   */
+  className?: string,
+  /**
+   * The component used for the root node.
+   * Either a string to use a DOM element or a component.
+   */
+  component?: string | ComponentType<*>,
+};
+
+type AllProps = DefaultProps & Props;
+
+class TableFooter extends React.Component<AllProps, void> {
```

Comment:
Is void needed? I thought that `React.Component<AllProps>` was enough. I could be wrong, I'm not much involved into flow.

Reply:
I'm pretty sure that _I_ didn't write that but just copied this or the prettier added this. :thinking: